lib/futuristic.dart

```diff
@@ -18,47 +18,45 @@ class Futuristic<T> extends StatefulWidget {
 
   /// Widget to display while the [Future] is executing.
   /// If null, a [CircularProgressIndicator] will be displayed.
-  final WidgetBuilder busyBuilder;
+  final WidgetBuilder? busyBuilder;
 
   /// Widget to display when the [Future] has completed with an error.
   /// If null, [initialBuilder] will be displayed again.
   /// The [Object] is the [Error] or [Exception] returned by the [Future].
   /// Call [VoidCallback] to start executing the [Future] again.
-  final Widget Function(BuildContext, Object, VoidCallback) errorBuilder;
+  final Widget Function(BuildContext, Object, VoidCallback)? errorBuilder;
 
   /// Widget to display when the [Future] has completed successfully.
   /// If null, [initialBuilder] will be displayed again.
-  final Widget Function(BuildContext, T) dataBuilder;
+  final Widget Function(BuildContext, T)? dataBuilder;
 
   /// Callback to invoke when the [Future] has completed successfully.
   /// Will only be invoked once per [Future] execution.
-  final ValueChanged<T> onData;
+  final ValueChanged<T>? onData;
 
   /// Callback to invoke when the [Future] has completed with an error.
   /// Will only be invoked once per [Future] execution.
   /// Call [VoidCallback] to start executing the [Future] again.
-  final Function(Object, VoidCallback) onError;
+  final Function(Object, VoidCallback)? onError;
 
   const Futuristic({
-    Key key,
-    @required this.futureBuilder,
+    Key? key,
+    required this.futureBuilder,
     this.autoStart = false,
-    this.initialBuilder,
+    required this.initialBuilder,
```

Comment:
The README and comment on line 16 should also be updated to reflect this change, or (better) initialBuilder should be nullable and optional to preserve the original behavior.